src/ComposerRequireChecker/NodeVisitor/DefinedSymbolCollector.php

```diff
@@ -60,6 +61,15 @@ private function recordClassDefinition(Node $node): void
         $this->recordDefinitionOf($node);
     }
 
+    private function recordEnumDefinition(Node $node): void
```

Comment:
Is a unit test possible here? :thinking:

Reply:
Added a unit test in `test/ComposerRequireCheckerTest/NodeVisitor/DefinedSymbolCollectorTest.php`